### PR TITLE
LPS-188842 portal-search-admin-web: Apply

### PR DESCRIPTION
**Pull request by gustavolimav**

### CODE REVIEW

File: modules/apps/portal-search/portal-search-admin-web/src/main/java/com/liferay/portal/search/admin/web/internal/display/context/builder/SearchAdminDisplayContextBuilder.java [hunk 1]
```diff
@@ -53,23 +52,21 @@ public SearchAdminDisplayContext build() {
...
 		if (_isIndexInformationAvailable() && permissionChecker.isOmniadmin()) {
 			_addNavigationItemList(
-				navigationItemList, "field-mappings", selectedTab);
+				navigationItemList, "field-mappings", "index-actions");
 		}
 
 		searchAdminDisplayContext.setNavigationItemList(navigationItemList);
 
-		searchAdminDisplayContext.setSelectedTab(selectedTab);
+		searchAdminDisplayContext.setSelectedTab("index-actions");
```
Comment (BryanEngler) [inline modules/apps/portal-search/portal-search-admin-web/src/main/java/com/liferay/portal/search/admin/web/internal/display/context/builder/SearchAdminDisplayContextBuilder.java]: im not understanding this logic change (the commit message just says SF?). why is "index-actions" always the selected tab now?

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-admin-web/src/main/java/com/liferay/portal/search/admin/web/internal/display/context/builder/SearchAdminDisplayContextBuilder.java]: I thing I made a wrong refractor 😅

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/constants/SearchAdminConstants.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.search.constants;
+
+/**
+ * @author Gustavo Lima
+ */
+public class SearchAdminConstants {
```
Comment (BryanEngler) [inline modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/constants/SearchAdminConstants.java:11]: i dont think i would do this... if i did add a constants class, i would probably add them to com.liferay.portal.search.admin.web.internal.constants, not portal-search-api

also, you only applied to SearchAdminDisplayContextBuilder, but left out view.jsp etc

unless BChan asked for constants, we can probably leave them out. this pr is pretty big already

**Comment by BryanEngler** [comment]
https://github.com/BryanEngler/liferay-portal/commit/be1c1ad135c21e1ae2b083af6e22afc6ed5d31e3

maybe try something like this for the selected tab logic

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/constants/SearchAdminConstants.java (hunk 2 above)
Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/constants/SearchAdminConstants.java:11]: Yeah I will do that, thx!

**Pull request closed without merging**